upload-cloud-storage/src/util.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Directory walk helper
+ * @param directory The path of the directory to traverse
+ * @param fileList  The files within the directory
+ */
+export async function getFiles(
```

Comment:
Is there a reason this function lives in its own file? I'm not seeing it being used outside of upload-helper.
Also, this function is missing unit test coverage.

Reply:
Added UT coverage. This function is just a helper for traversing the dir tree; I thought it made sense to keep it separate but happy to move into upload-helper if that makes more sense.